plugins/modules/ec2_vpc_net.py

```diff
@@ -242,21 +243,32 @@ def vpc_exists(module, vpc, name, cidr_block, multi):
     return None
 
 
-@AWSRetry.backoff(delay=3, tries=8, catch_extra_error_codes=['InvalidVpcID.NotFound'])
 def get_classic_link_with_backoff(connection, vpc_id):
     try:
-        return connection.describe_vpc_classic_link(VpcIds=[vpc_id])['Vpcs'][0].get('ClassicLinkEnabled')
+        results = connection.describe_vpc_classic_link(aws_retry=True, VpcIds=[vpc_id])
+        return results['Vpcs'][0].get('ClassicLinkEnabled')
     except is_boto3_error_message('The functionality you requested is not available in this region.'):
```

Comment:
In addition to the boto3_error_message, what do you think about adding also `except (botocore.exceptions.ClientError, botocore.exceptions.BotoCoreError) as e:`?